package.json

```diff
@@ -167,6 +167,16 @@
 					}
 				}
 			}
+		],
+		"keybindings": [
+			{
+				"command": "meadow.selectDevice",
+				"when": "editorTextFocus",
+				"mac": "cmd+shift+r",
```

Comment:
As much as I really want to have the same command across platforms, this is currently the shortcut for opening the Debug view (**Show Run and Debug** command). We might need a different one for VS Code use to avoid stomping on that [default](https://code.visualstudio.com/docs/getstarted/keybindings#_display).

![VS Code command picker with the View command Show Run and Debug highlighted with a shortcut of command, shift, D.](https://user-images.githubusercontent.com/713665/178156040-63cb58f3-a206-4486-9efa-7d113253743a.png)

[Ported over from #19]

Reply:
Thanks @patridge. I've temporarily changed it to CMD/Ctrl+Shift+R. At least on Mac that doesn't seem to clash with anything, but it needs further investigation. I'll update the docs to mention the shortcut.